src/System-Changes/ChangeSet.class.st

```diff
@@ -186,6 +186,21 @@ ChangeSet class >> doesAnyChangeSetHaveClass: aClass andSelector: aSelector [
 	^ (self countOfChangeSetsWithClass: aClass andSelector: aSelector) > 0
 ]
 
+{ #category : #'as yet unclassified' }
```

Comment:
Method is unclassified

Reply:
I categeorized only the classes - not the methods here